docs/documentation/powertrain_builder/pt_visual/sort_mechanism.rst

```diff
@@ -0,0 +1,105 @@
+.. _sort_mechanism:
+
+=================
+Hierarchy sorting
+=================
+The sorting mechanism inspired by the `Tutte's drawing algorithm <https://www.youtube.com/watch?v=mEzPPMhR8XE&t=45s>`_
+is utilized before each visualization in a `static html <https://www.geeksforgeeks.org/computer-networks/difference-between-static-and-dynamic-web-pages/>`_
```

Comment:
You don't have to specify about the static html imo you can jsut end the sentence with `utilized before each visualization.`